charts/lagoon-core/ci/linter-values.yaml

```diff
@@ -163,6 +163,67 @@ ssh:
   resources:
     requests:
       cpu: "10m"
+  hostKeys:
+    rsaPub: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDTU9JXClTTQQB2QvsNegwWOwxdZNyswjDrWO94c7Gej+WkeFC6iYgIkALTFXP+/uSgMdcxISnAqaFial7j2owmnGKbH9MTTmSS68I8FZRN/KAsDTVqzOR8bcwo36uS6cjcvQJevFpKJk/SWDMozqRgWK2I3QfkkCvGyRA9x/kq9BKFORjMzcbRnSTv3VPsXJgJSIQGYp+76TTNIT28h+WezGVx1Yf+sKmhJjgVnMcvim66KW+70fgeaScf5sgsWBX+31T5RkVqarhW+FBcHuFk5N3bmsG7k+894nssqDvVPXEKly386q4Z9OykaYv+q/o91wAA+UwUg7Pb+6D7X80cjxLBENoH2HSraKTj+Z0e6i7usCu2Zd/xUGUiSmLtfQ1hP83HeeBEWUeVijs70/gtV9Xj8GXncaKoKyV/Ws6w2nCK4b1UgfV0bMLDdgjeA0+KFA8RSckoVWYtFh3zUxG/vGeAVZkgncKShEJMgeQZJ9DzutoO2ZZsYTth1C8rab0="
```

Comment:
I don't think you need the public key. Just the private is enough for `HostKey`.

Reply:
If you don't have a matching public key, the error log spits out a complaint every single time!